e2e/benchmarks/browserstack-benchmark/app_node_test.js

```diff
@@ -313,20 +313,32 @@ describe('promise queue', () => {
     await promises[2];
     await promises[3];
     expect(started).toEqual(
-        [true, true, true, true, true]
+      [true, true, true, true, true]
     );
     expect(resolved).toEqual(
-        [true, true, true, true, false]
+      [true, true, true, true, false]
     );
 
     // The last promise should finish
     jasmine.clock().tick(1000);
     await promises[4];
     expect(started).toEqual(
-        [true, true, true, true, true]
+      [true, true, true, true, true]
     );
     expect(resolved).toEqual(
-        [true, true, true, true, true]
+      [true, true, true, true, true]
+    );
+  });
+});
+
+
+describe('weekly schedule', () => {
+  it('scheduling models works', () => {
+    const dt = new Date("October 19, 2022");  // Wednesday.
+    models = Array.from(Array(24).keys());
+    const res = scheduleModels(models, dt);
+    expect(res).toEqual(
+      [12, 13, 14, 15]
     );
   });
```

Comment:
Nit: Maybe add some tests for edge cases like day 0 (Sunday) and day 6 (Saturday), although I'm pretty sure your function is correct.

Reply:
Done.